src/main/java/spireMapOverhaul/zones/brokenSpace/relics/BrokenPrayerWheel.java

```diff
@@ -0,0 +1,49 @@
+package spireMapOverhaul.zones.brokenSpace.relics;
+
+import com.evacipated.cardcrawl.modthespire.lib.SpirePatch2;
+import com.evacipated.cardcrawl.modthespire.lib.SpirePrefixPatch;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.relics.AbstractRelic;
+import com.megacrit.cardcrawl.relics.PrayerWheel;
+import com.megacrit.cardcrawl.rewards.RewardItem;
+
+import static spireMapOverhaul.SpireAnniversary6Mod.makeID;
+
+public class BrokenPrayerWheel extends BrokenRelic {
+    public static final String ID = "BrokenPrayerWheel";
+    public static final int GOLD = 2;
+    public static final int AMOUNT = 1;
+
+
+    public BrokenPrayerWheel() {
+        super(ID, RelicTier.SPECIAL, LandingSound.CLINK, PrayerWheel.ID);
+
+    }
+
+    @Override
+    public int changeNumberOfCardsInReward(int numberOfCards) {
+        return numberOfCards - AMOUNT;
+    }
+
+    @Override
+    public String getUpdatedDescription() {
+        return DESCRIPTIONS[0] + AMOUNT + DESCRIPTIONS[1] + GOLD + DESCRIPTIONS[2];
+    }
+
+    @SpirePatch2(clz = RewardItem.class, method = "applyGoldBonus")
```

Comment:
This patch doubles the gold when `applyGoldBonus` is called, but that method can be called multiple times -- it's called both in the constructor and by the `incrementGold` method. This means that when someone else writes logic to add extra gold, your patch will double the gold an extra time.

You can fix this by changing this to a patch targeting the two RewardItem constructors that take in a `gold` parameter. You should be able to use `@ByRef` to change the `gold` variable.

Reply:
I realized what I suggested might have a different bug -- I think it will double the amount of gold when you save and reload. So ignore the suggestion solution in my previous comment.

What you can do instead is patch `applyGoldBonus`, but have this work the same way other gold bonuses (e.g. Golden Idol) work. They use the `bonusGold` field to store the extra gold, which avoids issues with repeatedly calling `applyGoldBonus` (since the bonus amount is recalculated based on the normal amount of gold). Looks like that will need an insert patch, but shouldn't be that bad.

This is what I should have thought of first, really. There are other ways you could set this up, but it's often the best bet to write code to work the same way that the base game does similar functionality. It makes compatibility a lot easier.